src/navigation/appNavigation.js

```diff
@@ -523,16 +514,6 @@ const changePinFlow = createStackNavigator({
   [CHANGE_PIN_CONFIRM_NEW_PIN]: ChangePinConfirmNewPinScreen,
 }, StackNavigatorModalConfig);
 
-// PARTICIPATE IN ICO FLOW
-const participateInICOFlow = createStackNavigator(
-  {
-    [ICO_PARTICIPATE]: ParticipateScreen,
-    [ICO_INSTRUCTIONS]: InstructionsScreen,
-    [ICO_CONFIRM]: ConfirmScreen,
-  },
-  StackNavigatorModalConfig,
-);
-
```

Comment:
+1 for bravery to remove this 😄

Reply:
:))
🧹🧹🧹🧹🧹🧹
_If we ever get back to this, I'm pretty sure we would have different UI_